service/controller/v13/templates/cloudconfig/decrypt_keys_assets_script.go

```diff
@@ -1,7 +1,95 @@
 package cloudconfig
 
 const DecryptKeysAssetsScript = `#!/bin/bash -e
+{{ if eq .EncrypterType "vault" }}
```

Comment:
There is a lot of bash magic here and I see what you mean about the boilerplate with similar logic in decrypt_tls_assets_script.go

@corest @calvix Could you help here as I think you'll have better input that me.

Reply:
Thanks, I'd like to get also input from @giantswarm/sig-security here.